### DISCUSSION TITLE
NCD to Latimer Core

### DISCUSSION THREAD

**Discussion by Jagullov** [tdwg/ltc#78] [Public Review]
Hi Everyone, thanks Latimer Core folks for inviting discussion here. I suppose it must be somewhere in the documentation, forgive me; but I can't find it. Is there a mapping between the NCD standard and Latimer Core fields?

Our Museum has a website https://www.bioexplora.cat/ncd/inici/lang-uk where it disseminates the description of its collection fund using the NCD standard and is considering to switch from the NCD standard to the LtC standard.

**Comment by jbstatgen** [comment]
Dear @Jagullov , 

Thank you very much for your interest in Latimer Core and your inquiry. It is exciting to see that you and colleagues are interested in learning about the standard to be able to apply and implement it. 

As task, now maintenance group we hadn't been aware that NCD had been implemented, and discussed your question as well as the availability of mappings between NCD and LtC during our last meeting. Please apologize that I'm a slow messenger and only answer now. The short answer to your question is that no up-to-date mapping exists. However, in the transition from NCD to the restart of its further development that led to LtC, NCD had been investigated in detail and included in mappings to - at that point - informal collection description terms. You can find those mappings as part of the archived (ie. read-only) TDWG-CD Github repository under [reference](https://github.com/tdwg/cd/tree/master/reference) in the folders [ncd](https://github.com/tdwg/cd/tree/master/reference/ncd) and [crosswalks](https://github.com/tdwg/cd/tree/master/reference/crosswalks). 

We are planning to organize open community sessions ("Happy Hours") every quarter for exchanges, questions & answers, and discussions of specific topics of interest. The first public session will be on Thursday June 13, 2024 (15 UTC). Inspired by your question, we are proposing the transition from NCD to LtC as topic. Might you be willing to introduce us to your implementation of NCD and explore with the group a possible mapping to LtC? 

We would also like to put out a call to everyone who used NCD as guidance or implemented it to join the Happy Hour and contribute their experiences to the discussion.

Again, thanks a lot for your inquiry into mappings from NCD to LtC. Let us know if you have further questions, and if you would be interested in joining the Happy Hour.

Best wishes,

Jutta

**Reply by Jagullov** [reply]
Thank you for your response Jutta, we will review the information and documents you have mentioned.

Our project was carried out in 2012 and we are currently evaluating how we can reformulate the project with a more up-to-date standard, thus taking the opportunity to give it a new impetus. We understand that LtC will be useful for this purpose, but we are just beginning to look at how we will approach the project. We will follow your progress.

Best regards,
Jordi
